cfme/base/ui.py

```diff
@@ -62,3 +109,612 @@ class Dashboard(CFMENavigateStep):
+
+
+@navigator.register(Server)
+class Authentication(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.authentication.select()
+
+
+@navigator.register(Server)
+class Workers(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.workers.select()
+
+
+@navigator.register(Server)
+class CustomLogos(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.customlogos.select()
+
+
+@navigator.register(Server)
+class Advanced(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.advanced.select()
+
+
+class ServerDiagnosticsView(ConfigurationView):
+    @View.nested
+    class summary(Tab):  # noqa
+        TAB_NAME = "Summary"
+
+    @View.nested
+    class workers(Tab):  # noqa
+        TAB_NAME = "Workers"
+
+    @View.nested
+    class collectlogs(Tab):  # noqa
+        TAB_NAME = "Collect Logs"
+
+    @View.nested
+    class cfmelog(Tab):  # noqa
+        TAB_NAME = "CFME Log"
+
+    @View.nested
+    class auditlog(Tab):  # noqa
+        TAB_NAME = "Audit Log"
+
+    @View.nested
+    class productionlog(Tab):  # noqa
+        TAB_NAME = "Production Log"
+
+    @View.nested
+    class utilization(Tab):  # noqa
+        TAB_NAME = "Utilization"
+
+    @View.nested
+    class timelines(Tab):  # noqa
+        TAB_NAME = "Timelines"
+
+    configuration = Dropdown('Configuration')
+
+
+@navigator.register(Server)
+class Diagnostics(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Configuration')
+
+    def step(self):
+        self.view.accordions.diagnostics.tree.click_path(
+            self.obj.zone.region.settings_string,
+            "Zone: {} (current)".format(self.obj.zone.description),
+            "Server: {} [{}] (current)".format(
+                self.obj.name, self.obj.sid))
+
+
+@navigator.register(Server)
+class DiagnosticsDetails(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.summary.select()
+
+
+@navigator.register(Server)
+class DiagnosticsWorkers(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.workers.select()
+
+
+class DiagnosticsCollectLogsView(ServerDiagnosticsView):
+    edit = Button(title="Edit the Log Depot settings for the selected Server")
+
+
+@navigator.register(Server)
+class DiagnosticsCollectLogs(CFMENavigateStep):
+    VIEW = DiagnosticsCollectLogsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.collectlogs.select()
+
+
+@navigator.register(Server)
+class DiagnosticsLogsSettings(CFMENavigateStep):
```

Comment:
This name was confusing when called on line 608, because it was missing the 'Collect' context. Consider DiagnosticsCollectLogsEdit?